.pipelines/cni/load-test-templates/validate-state-template.yaml

```diff
@@ -12,9 +13,19 @@ steps:
       scriptType: "bash"
       addSpnToEnvironment: true
       inlineScript: |
-        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
+        if [ ${{ parameters.cnsManagedEndpoint }} == "true" ] && [ ${{ parameters.restartCase }} == "true" ]
+        then
+          echo If this step fails, test manually.
+          echo Only fails when CNS is managing the endpoint state and state file does not existing due to node restart with 0 pods, outside of kube-system pods, scheduled on node.
+          echo Failure is due to slow node restart which occurs after scale down. This also misses the intent of the restart node scenario.
+          echo Timing should be: Scale down > Restart nodes during scale down > complete scale down > validate.

```

Comment:
how slow is too slow to restart? would it be possible to wait longer and avoid the manual testing?

Reply:
The scale down operation takes roughly 3 minutes to complete. If the restart does not happen/start in that timeframe then it would be considered too slow. As of now I do not know of a way to capture the status of a VMSS restart reliably. You get provisioned or not provisioned states along with a few other fields, but nothing really confirms how the restart is proceeding.

Ideally we could put a timeout fail after X amount of time. I am just not sure if we can put a time on what that is as the restart could happen correctly, but not complete for a while as well.